### DISCUSSION TITLE
How to load a local model？

### DISCUSSION THREAD

**Discussion by sulutian** [openai/whisper#1823] [Q&A]
How to load a local mode？

**Comment by phineas-pta** [comment]
u mean fine tuned model or something else ?

**Reply by warrenkc** [reply]
For example if I want to use this model: https://huggingface.co/cahya/whisper-large-id/tree/main

**Reply by phineas-pta** [reply]
it's fine tuned model

use the `transformers` package is easier

**Comment by afsara-ben** [comment]
but how to load it so that its wrapped in `WhisperForConditionalGeneration` class?
